googlefitmodule/src/main/java/com/livestrong/tracker/googlefitmodule/main/LSGoogleFitService.java

```diff
@@ -0,0 +1,86 @@
+package com.livestrong.tracker.googlefitmodule.main;
+
+import android.app.IntentService;
+import android.content.Intent;
+import android.util.Log;
+
+import java.text.SimpleDateFormat;;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by shambhavipunja on 1/26/16.
+ */
+public class LSGoogleFitService extends IntentService {
+    public static final String TAG = "Service";
+    private HashMap<Date,Integer> mStepMap;
```

Comment:
I would use interface instead of concrete class here. For instance, Map instead of HashMap.